docs/index.rst

```diff
@@ -18,79 +18,160 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-HOME
-====
+Welcome
+=======
+
+Apache Kyuubi™ is a distributed and multi-tenant gateway to provide serverless
+SQL on Data Warehouses and Lakehouses.
+
+Kyuubi builds distributed SQL query engines on top of various kinds of modern
+computing frameworks, e.g.,
+Apache `Spark <https://spark.apache.org/>`_,
+`Flink <https://flink.apache.org/>`_,
+`Doris <https://doris.apache.org/>`_,
+`Hive <https://hive.apache.org/>`_,
```

Comment:
Is it necessary to add Apache in front of Flink, Doris, and Hive, considering they are Apache's trade mark? Or just remove all the `Apache` in this list including for Spark.

Reply:
It's ok to keep it as-is

Reply:
alright